css/components/admin/widget/metadata_form.scss

```diff
@@ -0,0 +1,18 @@
+.c-widget-metadata-form {
+  
+  .remove-widget-link-container {
```

Comment:
We would avoid this kind of style by using a component, but it's your call....

Reply:
👍